packages/lisk-p2p/src/response_handler.ts

```diff
@@ -69,10 +68,35 @@ export const instantiatePeerFromResponse = (peer: unknown): PeerConfig => {
 		ipAddress: rawPeer.ip,
 		wsPort,
 		height,
-		id,
 		os,
 		version,
 	};
 
 	return peerConfig;
 };
+
+interface RPCPeerListResponse {
+	readonly peers: ReadonlyArray<object>;
+	readonly success?: boolean; // Could be used in future
+}
+
+export const processPeerListFromResponse = (
+	response: unknown,
+): ReadonlyArray<PeerConfig> => {
+	if (!response) {
+		return [];
```

Comment:
if response is undefined, shouldnt' this throw error?
empty response should never happen, and it should be handle by developer?